addon/components/power-select-with-create.js

```diff
@@ -40,7 +40,10 @@ export default Ember.Component.extend({
 
       if (this.get('search')) {
         return Ember.RSVP.resolve(this.get('search')(term)).then((results) =>  {
-          results.unshift(this.buildSuggestionForTerm(term));
+          let shouldShowCreateOption = this.get('showCreateWhen') ? this.get('showCreateWhen')(term) : true;
```

Comment:
Since it is now used twice, can you refactor this into something like

```js
if (this.shouldShowCreateOption(term)) {
  results.unshift(this.buildSuggestionForTerm(term));
}
```